airflow/providers/amazon/aws/operators/athena.py

```diff
@@ -39,9 +39,7 @@ class AWSAthenaOperator(BaseOperator):
     :param output_location: s3 path to write the query results into. (templated)
     :type output_location: str
     :param aws_conn_id: aws connection to use
-    :type aws_conn_id: str
-          (adding this param to template_fields so that it can be overriden using jinja template from Dags
-           This feature can be useful if user wants to update/override the aws_conn_id for some kind of Dag Isolation etc) 
+    :type aws_conn_id: <str> adding this param to template_fields so that it can be overriden using jinja template from Dags 
```

Comment:
Thanks for the update! But this is still missing the mark a bit.

1. You  are still updating the `type` line. The type of this input is **not changing**. Please **do not** update this line. Instead add your documentation to the `param` line.
2. You are still using an "action word" here: "**Adding** this param to template_fields so that ...". This type of verbiage only makes sense in something like a git commit message or PR description.

I'll echo the same recommendation I made previously, please update the `:param aws_conn_id` line. You can see the above lines what the comments look like for templated fields. They have simply added `(templated)` to the end of the doc string line. Or if you really want more details you could perhaps add something like:

>  param aws_conn_id: aws connection to use. This param can also be overridden using Jinja templating from DAGs.

Reply:
Please make the same updates to the other modules as well.

Reply:
sorry about the confusion. updated the requested changes again for all the modules. Please validate and let me know if anything is missed.